test/conformance/testing/include/uur/fixtures.h

```diff
@@ -993,6 +993,22 @@ struct urKernelExecutionTest : urKernelTest {
     std::vector<ur_mem_handle_t> buffer_args;
     uint32_t current_arg_index = 0;
 };
+
+template <class T> struct GlobalVar {
+    std::string name;
+    T value;
+};
+
+struct urGlobalVariableTest : uur::urKernelExecutionTest {
+    void SetUp() override {
+        program_name = "device_global";
+        global_var = {"_Z7dev_var", 0};
```

Comment:
The mangled name here I got from inspecting the SYCL_PI_TRACE with the call - perhaps we could do something similar to the `kernel_names` header here - but just kept it simple for now.

Reply:
makes sense to me